www/gatsby-node.js

```diff
@@ -268,7 +268,12 @@ exports.createPages = ({ graphql, actions, reporter }) => {
 
   createRedirect({
     fromPath: `/docs/advanced-tutorials/`,
-    toPath: `/tutorial/advanced-tutorials/`,
+    toPath: `/tutorial/additional-tutorials/`,
+    isPermanent: true,
+  })
+  createRedirect({
+    fromPath: `/docs/advanced-tutorials/`,
```

Comment:
```suggestion
    fromPath: `/tutorial/advanced-tutorials/`,
```

I think you mean this?

Reply:
Good catch! I think that migraine took over my cognition when I was finishing the PR...and I forgot to update the copy and paste.